dashboards-observability/public/components/event_analytics/explorer/explorer.scss

```diff
@@ -21,4 +21,15 @@
 .mainContentTabs .euiResizableContainer {
   height: calc(100vh - 298px);
 }
-  
\ No newline at end of file
+  
+.data_config_resizable_btn {
+  margin-left: 2px !important;
```

Comment:
Please don't use !important as it is proven to non easily debuggable issues, let's remove those '!important ' and try to use specificity hierarchy to achieve the same effect.